packages/terra-html-table/src/TableBody.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  /**
+   * The children passed to the component
+   */
+  children: PropTypes.node,
+};
+
+const defaultProps = {
+  children: [],
```

Comment:
Should we default t0 [] or should we render null if children are null? How does this affect accessibility of composition if no children are supplied?